README.md

```diff
@@ -472,13 +612,71 @@ Supported operators are `and`, `or` and `xor`.
 
 More examples are available in [`tests/assertions_operators.yml`](/tests/assertions_operators.yml).
 
+# Write and run your first test suite 
+
+To try to understand how the Venom works, we will create and run a first testsuite together.
+
+The first test that we will do, in this testsuite, is to test whether the site we want to test (a public REST API, OVHcloud API for example):
+- is accessible (respond with a 200 status code)
+- respond quickly (less than 1 second)
+- if its answer is not empty
```

Comment:
```suggestion
```

Reply:
Avec l'assertion "result.bodyjson ShouldContainKey apis" je regarde si la réponse n'est pas vide car je regarde si elle contient "apis"

Reply:
I think the last point is enough to describe what you are trying to achieve.